examples/grpc-service/src/main/java/example/armeria/grpc/HelloServiceImpl.java

```diff
@@ -0,0 +1,140 @@
+package example.armeria.grpc;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.concurrent.TimeUnit;
+
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+import example.armeria.grpc.Hello.HelloReply;
+import example.armeria.grpc.Hello.HelloRequest;
+import example.armeria.grpc.HelloServiceGrpc.HelloServiceImplBase;
+import io.grpc.stub.StreamObserver;
+import reactor.core.publisher.Flux;
+
+public class HelloServiceImpl extends HelloServiceImplBase {
+
+    /**
+     * Sends an {@link HelloReply} immediately when receiving a request.
+     */
+    @Override
+    public void hello(HelloRequest request, StreamObserver<HelloReply> responseObserver) {
+        responseObserver.onNext(buildReply(toMessage(request.getName())));
+        responseObserver.onCompleted();
+    }
+
+    /**
+     * Sends an {@link HelloReply} 3 seconds after receiving a request.
+     */
+    @Override
+    public void lazyHello(HelloRequest request, StreamObserver<HelloReply> responseObserver) {
+        // You can use the event loop for scheduling.
+        RequestContext.current().contextAwareEventLoop().schedule(() -> {
+            responseObserver.onNext(buildReply(toMessage(request.getName())));
+            responseObserver.onCompleted();
+        }, 3, TimeUnit.SECONDS);
+    }
+
+    /**
+     * Sends an {@link HelloReply} using {@code blockingTaskExecutor}.
+     *
+     * @see <a href="https://line.github.io/armeria/server-grpc.html#blocking-service-implementation">
+     *      Blocking service implementation</a>
```

Comment:
How about breaking after `Blocking` so that no space is added between `>` and `Blocking`?

```
@see <a href="https://line.github.io/armeria/server-grpc.html#blocking-service-implementation">Blocking
     service implementation</a>
```